packages/core/src/utils.ts

```diff
@@ -1056,3 +1056,12 @@ export function append<T>(array: T[] | undefined, value: T): T[] {
 export function getWebSocketUrl(path: string, baseUrl: URL | string): string {
   return new URL(path, baseUrl).toString().replace('http://', 'ws://').replace('https://', 'wss://');
 }
+
+/**
+ * Sorts an array of strings using the localeCompare method.
```

Comment:
Can we also note that it mutates and returns the input array?

Reply:
✅  comments updated